web/styles/compose.css

```diff
@@ -329,9 +329,16 @@
 
     &:has(.new_message_textarea.over_limit),
     &:has(.new_message_textarea.over_limit:focus) {
+        border-color: hsl(0deg 76% 65%);
```

Comment:
These border-color values would also be great expressed as CSS variables. Same with the box-shadow color below.